src/lib/swapper/swappers/ThorchainSwapper/utils/getSignTxFromQuote.ts

```diff
@@ -41,11 +40,11 @@ export const getSignTxFromQuote = async ({
   from,
   xpub,
   supportsEIP1559,
+  slippageTolerancePercentage,
 }: GetSignTxFromQuoteArgs): Promise<UnsignedTx> => {
   const { recommendedSlippage } = quote
 
-  const slippageTolerance =
-    recommendedSlippage ?? getDefaultSlippagePercentageForSwapper(SwapperName.Thorchain)
+  const slippageTolerance = slippageTolerancePercentage ?? recommendedSlippage
```

Comment:
i dont know of any place where recommendedSlippage is set at all.... it would be better to just remove it

Reply:
Recommended slippage is currently only set on ThorSwap quotes (but it is indeed set here), where we receive a recommended value from their API based on the size of the trade and asset pair.

See https://github.com/shapeshift/web/blob/9e8ff16a0967a35e1b93d6eec774cb3d87534892/src/lib/swapper/swappers/ThorchainSwapper/getThorTradeQuote/getTradeQuote.ts#L110